src/installation/install_epinio_on_rancher_desktop.md

```diff
@@ -19,8 +21,20 @@ Make sure Rancher Desktop is running.
 Rancher Desktop can report Kubernetes as running while some pods are actually not yet ready.
 Manual verification is possible by executing the command `kubectl get pods -A` in a terminal and checking that all pods report either `Running` or `Completed` as their status.
 
-Follow [Installation using a Custom Domain](./install_epinio_customDNS.md) for test and production environments.
+For now, Rancher Desktop configures his own loadbalancer (Traefik through K3s) and you have to set Epinio system domain according to the provided IP, for example:
+```
+LB_IP=$(kubectl describe svc traefik --namespace traefik \
+        | awk '/Ingress/ { print $3 }')
+
+export EPINIO_SYSTEM_DOMAIN=${LB_IP}.omg.howdoi.website
+```
+Previous commands are for Bash, but can be easily adapted for Windows shell.
+
+If you don't want to use the default "magic" DNS please follow [Install Epinio with "magic" DNS](./install_epinio_magicDNS.md).
+
+The Epinio installation is pretty much identical on Linux, MacOS and Windows:
+1. Start a terminal, use `cmd` or `powershell` on Windows (latest one is preferred) and your preferred one on Linux/MacOS (`bash` is the default one).
 
-### Linux
+2. Download the [latest version](https://github.com/epinio/epinio/releases) of Epinio binary and copy it on your client. It is recommended to add it to the `PATH` variable (`export PATH=<epinio-binary-path>:$PATH` on Linux/MacOS with Bash and on Windows please follow [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)).
 
-Linux is not yet supported by Rancher Desktop - see https://github.com/rancher-sandbox/rancher-desktop/issues/426
+3. Use `helm` to install Epinio by following either the [Method 1](./install_epinio_auto.md) (recommended for most cases) or the [Method 2](./install_epinio_manual.md) (useful for advanced configuration or if you already have some components in your environment). As **Traefik** is already installed, just bypass this step (by adding option `--set skipTraefik=true` for Method 1).
```

Comment:
I would mention the current issue we have on Linux until it will be fixed, WDYT?

Reply:
Done!